mne/preprocessing/ica.py

```diff
@@ -786,7 +786,7 @@ def _export_info(self, info, container, add_channels):
         ch_names = []
         ch_info = info['chs'] = []
         for ii in range(self.n_components_):
-            this_source = 'ICA %03d' % (ii + 1)
+            this_source = 'ICA %03d' % (ii)
```

Comment:
@mmagnuski You mean that we should ensure consistency across mne with regards to IC plots, yes? Not that we should 0-index the naming for all channel types. I am a fan of 0-indexing everything but I guess other channel names are one-based indexed for a reason.

A great accidental reference from @agramfort